test/WebJobs.Extensions.Http.Tests/HttpTestHelpers.cs

```diff
@@ -7,15 +7,21 @@
 using System.Text;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Features;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.Azure.WebJobs.Extensions.Tests.Extensions.Http
 {
     public class HttpTestHelpers
     {
         public static HttpRequest CreateHttpRequest(string method, string uriString, IHeaderDictionary headers = null, string body = null)
         {
+            var context = new DefaultHttpContext();
```

Comment:
Required for POST requests with a body. The http extension code is accessing these services, and we weren't mocking this out fully.